packages/framework-webpack/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@muban/storybook-framework-webpack",
+  "version": "8.0.0",
+  "description": "Storybook for Muban: The webpack framework.",
+  "keywords": [
+    "muban",
+    "storybook",
+    "webpack",
+    "framework"
+  ],
+  "author": "Arjan van Wijk",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/@mubanjs/muban-storybook.git"
+  },
+  "bugs": {
+    "url": "https://github.com/@mubanjs/muban-storybook/issues"
+  },
+  "homepage": "https://github.com/mubanjs/muban-storybook/tree/main/packages/framework-webpack#readme",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "types": "./dist/index.d.ts"
+    },
+    "./preset": {
+      "import": "./dist/preset.mjs",
+      "types": "./dist/preset.d.ts"
+    },
+    "./package.json": "./package.json"
+  },
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist/**/*",
+    "README.md",
+    "*.js",
+    "*.d.ts"
+  ],
+  "scripts": {
+    "dev": "npm run build -- --watch",
+    "build": "tsc -p ./tsconfig.json",
+    "typecheck": "tsc --project tsconfig.json --noEmit --noUnusedLocals",
```

Comment:
Is there a reason to only enable `--noUnusedLocals` in the typecheck command?

Reply:
This has been done for a while in all setups, and I think it was done that way to don't show errors in your IDE or build when you are busy writing code (and you might end up with unused vars temporarily), but do have it there when you typecheck / commit.

Alternatively you could have an extra tsconfig.json, but for just this flag (or two) this seems managable.